### ISSUE TITLE
Godot 3.1 crashes on project startup !

### ISSUE THREAD

**Issue by mohamedhany4116** [godotengine/godot#22790]
Hello , Godot developers I recently experienced an issue with the new Godot 3.1 alpha 
I am not entirely sure if this issue is from my computer or not but whenever I start godot
The main menu is just fine but whenever I start a project it shows me the Godot logo and 
For a few seconds and then it crashes !

My OS :
Linux mint cinnamon 18.3 32- bit

Godot version I have experienced the issue with :
Godot _v3.1-alpha1_x11.32

OpenGL version on my computer :
OpenGL 2.1

**Comment by swenner** [comment]
Please attach the commandline output and/or a backtrace.

**Comment by mohamedhany4116** [comment]
I am sorry I don't know how to access the command line output but I will try figuring out how to access it

**Comment by Zylann** [comment]
@mohamedhany4166 is it happening with any project? even empty?

**Comment by 5N08** [comment]
Sorry guys, should I start new issue since I have different OS?

**Comment by qarmin** [comment]
@5N08 no, if it same issue. Could you attach a log or video with crash?

**Comment by 5N08** [comment]
@qarmin Do you mean like the 3rd comment in this thread?

**Comment by mohamedhany4116** [comment]
@swenner I am so sorry for not replying or responding to anything but I have been way too busy for the last few months anyways II figured out how to access Godot from terminal. And when I launched Godot from terminal it gave me this Log

mohamedhany@mohamedhany-Invalid-entry-length-0-DMI-table-is-broken-Stop ~ $ godot
ERROR: initialize: Condition ' ctxErrorOccurred || !p->glx_context ' is true. returned: ERR_UNCONFIGURED
   At: platform/x11/context_gl_x11.cpp:188.
OpenGL ES 2.0 Renderer: Gallium 0.4 on ATI RS480
ERROR: _gl_debug_print: GL ERROR: Source: OpenGL	Type: Error	ID: 1	Severity: High	Message: GL_INVALID_OPERATION in glTexImage2D(bad target for texture)
   At: drivers/gles2/rasterizer_gles2.cpp:111.
ERROR: _gl_debug_print: GL ERROR: Source: OpenGL	Type: Error	ID: 1	Severity: High	Message: GL_INVALID_OPERATION in glTexImage2D(bad target for texture)
   At: drivers/gles2/rasterizer_gles2.cpp:111.
ERROR: _gl_debug_print: GL ERROR: Source: OpenGL	Type: Error	ID: 1	Severity: High	Message: GL_INVALID_OPERATION in glTexImage2D(bad target for texture)
   At: drivers/gles2/rasterizer_gles2.cpp:111.
ERROR: _gl_debug_print: GL ERROR: Source: OpenGL	Type: Error	ID: 1	Severity: High	Message: GL_INVALID_OPERATION in glTexImage2D(bad target for texture)
   At: drivers/gles2/rasterizer_gles2.cpp:111.
ERROR: _gl_debug_print: GL ERROR: Source: OpenGL	Type: Error	ID: 1	Severity: High	Message: GL_INVALID_OPERATION in glTexImage2D(bad target for texture)
   At: drivers/gles2/rasterizer_gles2.cpp:111.
ERROR: _gl_debug_print: GL ERROR: Source: OpenGL	Type: Error	ID: 1	Severity: High	Message: GL_INVALID_OPERATION in glTexImage2D(bad target for texture)
   At: drivers/gles2/rasterizer_gles2.cpp:111.
ERROR: _gl_debug_print: GL ERROR: Source: OpenGL	Type: Error	ID: 1	Severity: High	Message: GL_INVALID_OPERATION in glTexImage2D(bad target for texture)
   At: drivers/gles2/rasterizer_gles2.cpp:111.
ERROR: _gl_debug_print: GL ERROR: Source: OpenGL	Type: Error	ID: 1	Severity: High	Message: GL_INVALID_OPERATION in glTexImage2D(bad target for texture)
   At: drivers/gles2/rasterizer_gles2.cpp:111.
ERROR: _gl_debug_print: GL ERROR: Source: OpenGL	Type: Error	ID: 1	Severity: High	Message: GL_INVALID_OPERATION in glTexImage2D(bad target for texture)
   At: drivers/gles2/rasterizer_gles2.cpp:111.
ERROR: _gl_debug_print: GL ERROR: Source: OpenGL	Type: Error	ID: 1	Severity: High	Message: GL_INVALID_OPERATION in glTexImage2D(bad target for texture)
   At: drivers/gles2/rasterizer_gles2.cpp:111.
ERROR: _gl_debug_print: GL ERROR: Source: OpenGL	Type: Error	ID: 1	Severity: High	Message: GL_INVALID_OPERATION in glTexImage2D(bad target for texture)
   At: drivers/gles2/rasterizer_gles2.cpp:111.
ERROR: _gl_debug_print: GL ERROR: Source: OpenGL	Type: Error	ID: 1	Severity: High	Message: GL_INVALID_OPERATION in glTexImage2D(bad target for texture)
   At: drivers/gles2/rasterizer_gles2.cpp:111.
ERROR: _gl_debug_print: GL ERROR: Source: OpenGL	Type: Error	ID: 1	Severity: High	Message: GL_INVALID_OPERATION in glTexImage2D(bad target for texture)
   At: drivers/gles2/rasterizer_gles2.cpp:111.
ERROR: _gl_debug_print: GL ERROR: Source: OpenGL	Type: Error	ID: 1	Severity: High	Message: GL_INVALID_OPERATION in glTexImage2D(bad target for texture)
   At: drivers/gles2/rasterizer_gles2.cpp:111.
ERROR: _gl_debug_print: GL ERROR: Source: OpenGL	Type: Error	ID: 1	Severity: High	Message: GL_INVALID_OPERATION in glTexImage2D(bad target for texture)
   At: drivers/gles2/rasterizer_gles2.cpp:111.
ERROR: _gl_debug_print: GL ERROR: Source: OpenGL	Type: Error	ID: 1	Severity: High	Message: GL_INVALID_OPERATION in glTexImage2D(bad target for texture)
   At: drivers/gles2/rasterizer_gles2.cpp:111.
ERROR: _gl_debug_print: GL ERROR: Source: OpenGL	Type: Error	ID: 1	Severity: High	Message: GL_INVALID_OPERATION in glTexImage2D(bad target for texture)
   At: drivers/gles2/rasterizer_gles2.cpp:111.
ERROR: _gl_debug_print: GL ERROR: Source: OpenGL	Type: Error	ID: 1	Severity: High	Message: GL_INVALID_OPERATION in glTexImage2D(bad target for texture)
   At: drivers/gles2/rasterizer_gles2.cpp:111.
ERROR: _gl_debug_print: GL ERROR: Source: OpenGL	Type: Error	ID: 1	Severity: High	Message: GL_INVALID_OPERATION in glTexImage2D(bad target for texture)
   At: drivers/gles2/rasterizer_gles2.cpp:111.
ERROR: _gl_debug_print: GL ERROR: Source: OpenGL	Type: Error	ID: 1	Severity: High	Message: GL_INVALID_OPERATION in glTexImage2D(bad target for texture)
   At: drivers/gles2/rasterizer_gles2.cpp:111.
ERROR: _gl_debug_print: GL ERROR: Source: OpenGL	Type: Error	ID: 1	Severity: High	Message: GL_INVALID_OPERATION in glTexImage2D(bad target for texture)
   At: drivers/gles2/rasterizer_gles2.cpp:111.
ERROR: _gl_debug_print: GL ERROR: Source: OpenGL	Type: Error	ID: 1	Severity: High	Message: GL_INVALID_OPERATION in glTexImage2D(bad target for texture)
   At: drivers/gles2/rasterizer_gles2.cpp:111.
ERROR: _gl_debug_print: GL ERROR: Source: OpenGL	Type: Error	ID: 1	Severity: High	Message: GL_INVALID_OPERATION in glTexImage2D(bad target for texture)
   At: drivers/gles2/rasterizer_gles2.cpp:111.
ERROR: _gl_debug_print: GL ERROR: Source: OpenGL	Type: Error	ID: 1	Severity: High	Message: GL_INVALID_OPERATION in glTexImage2D(bad target for texture)
   At: drivers/gles2/rasterizer_gles2.cpp:111.
ERROR: _gl_debug_print: GL ERROR: Source: OpenGL	Type: Error	ID: 1	Severity: High	Message: GL_INVALID_OPERATION in glTexImage2D(bad target for texture)
   At: drivers/gles2/rasterizer_gles2.cpp:111.
ERROR: _gl_debug_print: GL ERROR: Source: OpenGL	Type: Error	ID: 1	Severity: High	Message: GL_INVALID_OPERATION in glTexImage2D(bad target for texture)
   At: drivers/gles2/rasterizer_gles2.cpp:111.
ERROR: _gl_debug_print: GL ERROR: Source: OpenGL	Type: Error	ID: 1	Severity: High	Message: GL_INVALID_OPERATION in glTexImage2D(bad target for texture)
   At: drivers/gles2/rasterizer_gles2.cpp:111.
ERROR: _gl_debug_print: GL ERROR: Source: OpenGL	Type: Error	ID: 1	Severity: High	Message: GL_INVALID_OPERATION in glTexImage2D(bad target for texture)
   At: drivers/gles2/rasterizer_gles2.cpp:111.
ERROR: _gl_debug_print: GL ERROR: Source: OpenGL	Type: Error	ID: 1	Severity: High	Message: GL_INVALID_OPERATION in glTexImage2D(bad target for texture)
   At: drivers/gles2/rasterizer_gles2.cpp:111.
ERROR: _gl_debug_print: GL ERROR: Source: OpenGL	Type: Error	ID: 1	Severity: High	Message: GL_INVALID_OPERATION in glTexImage2D(bad target for texture)
   At: drivers/gles2/rasterizer_gles2.cpp:111.
ERROR: _gl_debug_print: GL ERROR: Source: OpenGL	Type: Error	ID: 1	Severity: High	Message: GL_INVALID_VALUE in glTexImage2D(invalid width or height or depth)
   At: drivers/gles2/rasterizer_gles2.cpp:111.
ERROR: initialize: Directional shadow framebuffer status invalid
   At: drivers/gles2/rasterizer_scene_gles2.cpp:2474.

**Comment by Zireael07** [comment]
> OpenGL ES 2.0 Renderer: Gallium 0.4 on ATI RS480

Does this card even support OpenGL 2? The only link on Google that seems to make sense cannot be opened as it throws a misconfigured website warning...

**Comment by mohamedhany4116** [comment]
@Zireael07 yes it does I can run Godot 2.1.5 on it pretty smoothly with no issues or errors also it's an ATI Radeon Xpress 1150  I know its 12 Years old but I think that it can run Godot since Godot 3.1 supports OpenGL 2.1 and my video card Supports OpenGL 2.1

**Comment by Calinou** [comment]
@mohamedhany4116 Please try following the workaround described in https://github.com/godotengine/godot/issues/21551.

**Comment by mohamedhany4116** [comment]
@Calinou Thank you for trying to help me but I just can't figure it out so I give up and I am going to buy a new PC I can't find a more simpler solution than this.

**Comment by mohamedhany4116** [comment]
@Calinou Thank You So Much for helping me out now Godot 3.1 runs without crashing I have been trying to fix this for many months thank you so much for your help !
